packages/cli/src/controllers/users.controller.ts

```diff
@@ -329,73 +327,47 @@ export class UsersController {
 
 	@Patch('/:id/role')
 	@RequireGlobalScope('user:changeRole')
-	async changeRole(req: UserRequest.ChangeRole) {
+	async changeGlobalRole(req: UserRequest.ChangeRole) {
 		const {
-			MISSING_NEW_ROLE_KEY,
-			MISSING_NEW_ROLE_VALUE,
+			INVALID_PAYLOAD,
 			NO_ADMIN_ON_OWNER,
-			NO_USER_TO_OWNER,
 			NO_USER,
 			NO_OWNER_ON_OWNER,
 			NO_ADMIN_IF_UNLICENSED,
 		} = UsersController.ERROR_MESSAGES.CHANGE_ROLE;
 
-		const { newRole } = req.body;
-
-		if (!newRole) {
-			throw new BadRequestError(MISSING_NEW_ROLE_KEY);
-		}
-
-		if (!newRole.name || !newRole.scope) {
-			throw new BadRequestError(MISSING_NEW_ROLE_VALUE);
+		const { roleName: newRole } = req.body;
+		if (!newRole || !['member', 'admin'].includes(newRole)) {
```

Comment:
Let's use `class-validator` instead?

Reply:
I'm working on a global validation middleware that uses `class-validator`, so this will be soon replaced by that.